spring-integration-jms/src/main/java/org/springframework/integration/jms/AbstractJmsChannel.java

```diff
@@ -23,9 +23,17 @@
 import org.springframework.util.Assert;
 
 /**
+ * A base {@link AbstractMessageChannel} implementation for JMS-backed message channels.
+ *
+ * <p>When producer and consumer for a JMS destination are in the same application
+ * and process, instead of using a pair of inbound and outbound adapters, it is also
+ * possible to use a single message channel.
+ *
```

Comment:
This blank line is good.
And I really like to divide different JavaDoc sections via blank line.

This way, that `@since 2.0` section has to be surrounded with blank lines.
So, in the end `@author` section is separate, `@since` on its own. And then `@see` is nicely separated as well.